ui/src/app/app-routing.module.ts

```diff
@@ -71,9 +71,9 @@ const routes: Routes = [
         component: HomepageComponent,
         data: {title: 'Homepage'},
       }, {
-        path: 'data-use-agreement',
+        path: 'data-code-of-conduct',
         component: DataUseAgreementComponent,
```

Comment:
we should change the name of the component, to match

Reply:
Yeah i was not sure if thats taken care in the Update DUA story, i can make the changes here but it will be half i would prefer to change the name of file and everything if i am changing the name of the component. What do you think?

Reply:
Your change looks good to me